website/docs/reference/node-selection/test-selection-examples.md

```diff
@@ -95,16 +103,36 @@ dbt build --select "orders"
 
 <TabItem value="cautious" label="Cautious mode">
 
-It is possible to prevent tests from running if one or more of its parents is unselected (and therefore unbuilt); we call this "cautious" indirect selection.
+For example, only the "orders" model (or node) and its dependent models will be selected for testing or building:
 
-It will only include tests whose references are each within the selected nodes.
+```shell
+dbt test --select "orders" --indirect-selection=cautious
+dbt build --select "orders" --indirect-selection=cautious
+
+```
+
+</TabItem>
+
+<TabItem value="buildable" label="Buildable mode">
+
+For example, dbt runs tests that reference "orders" within the selected nodes (or their ancestors).
```

Comment:
This looks accurate 👍 

But I'd still suggest clarifying that the above statement doesn't apply to dbt in general, and _only_ applies to the example below:

```suggestion
In this example, dbt executes tests that reference "orders" within the selected nodes (or their ancestors).
```